### DISCUSSION TITLE
Objectron Dataset

### DISCUSSION THREAD

**Discussion by vlomonaco** [ContinualAI/avalanche#427] [Feature Request]
on the possibility of adding the Objectron Dataset in Avalanche: https://ai.googleblog.com/2020/11/announcing-objectron-dataset.html

**Comment by lrzpellegrini** [comment]
This may be a good starting point: https://github.com/google-research-datasets/Objectron/blob/master/notebooks/Objectron_Pytorch_tutorial.ipynb
